core/src/scene/sceneLoader.cpp

```diff
@@ -94,21 +94,12 @@ bool SceneLoader::applyUpdates(const std::shared_ptr<Platform>& platform, Scene&
         }
 
         if (value) {
-            try {
-                // Dummy node to trigger YAML exception on YAML syntax errors
-                auto parse = YAML::Load(update.path);
```

Comment:
I think we will not hit a yaml syntax error without this case. Its good to inform the user of this error.